mysql51/src/main/java/mysql51/badclassname.java

```diff
@@ -0,0 +1,24 @@
+package mysql51;
+
+public class badclassname {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename this class name to match the regular expression '^[A-Z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00101)